azure-pipelines.yml

```diff
@@ -235,6 +238,7 @@ jobs:
                  bin/py.test  -vvs tests/typecode \
                     tests/extractcode \
                     tests/plugincode \
+                    tests/cluecode \
```

Comment:
It would be best to be consistent and make cluecode always in the same position on all such blocks? In some case it comes last in some other case, next to last